h/cli/commands/devdata.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import click
+
+from h import models
+
+
+def create_all_dev_data(request):
+    """Create all the dev data that doesn't exist yet."""
+    factory = DevDataFactory(request)
+
+    # Create some Hypothesis users: an admin user, a staff user, a normal user, ...
+    factory.user(u"user")
+    factory.user(u"admin", admin=True)
+    factory.user(u"staff", staff=True)
+
+    # Create the auth client that the Hypothesis client needs to login.
+    factory.auth_client(u"a77b3fee-31e8-11e8-b65b-cf64c6731003",
+                        name=u"client",
+                        grant_type=u"authorization_code",
+                        response_type=u"code",
+                        redirect_uri=u"http://localhost:5000/app.html")
+
+    # Create auth clients, users etc for the publisher account test site.
+    partner_authority = u"partner.org"
+
+    factory.auth_client(u"e2317c70-3cce-11e8-ac47-23c1999ab55a",
+                        authority=partner_authority,
+                        name=u"publisher_account_test_site_client_credentials",
+                        grant_type=u"client_credentials",
+                        response_type=None,
+                        redirect_uri=None,
+                        secret=u"8l1b3f4vkS80k83WEMl47DKfsOraiQk4aHld-gNYOqM")
+    factory.auth_client(u"c78ed9e6-3cd1-11e8-b64e-535bf04ec9db",
+                        authority=partner_authority,
+                        name=u"publisher_account_test_site_client_credentials",
+                        grant_type=u"jwt_bearer",
+                        response_type=None,
+                        redirect_uri=None,
+                        secret=u"Zsid_Z1DVD9U-3V9bUN4008QNg9fkjm6jkI0tHIzK6Y")
+
+    publisher_site_admin = factory.user(authority=partner_authority,
+                                        username=u"admin")
+    publisher_site_organization = factory.organization(u"publisher_account_test",
+                                                       partner_authority)
+
+    factory.group(type_="open",
+                  name=u"Partner",
+                  creator=publisher_site_admin,
+                  origins=[u"http://localhost:5050"],
+                  organization=publisher_site_organization)
+
+    # Open and restricted groups for the BioPub demo site.
+    biopub_admin = factory.user(username=u"biopub_admin")
+    biopub_organization = factory.organization(u"BioPub")
+    factory.group(type_="open",
+                  name=u"BioPub Open",
+                  creator=biopub_admin,
+                  origins=[u"http://localhost:9000"],
+                  organization=biopub_organization)
+    factory.group(type_="restricted",
+                  name=u"BioPub Restricted",
+                  creator=biopub_admin,
+                  origins=[u"http://localhost:9000"],
+                  organization=biopub_organization)
+
+    request.tm.commit()
+
+
+class DevDataFactory(object):
```

Comment:
I share Lyza's concern that this is a lot of code and will need to be maintained. I wonder if there is a way to simplify it such that it's just a list of commands to create the env rather than this whole extra helper class in between?